### ISSUE TITLE
Ruby 2.5 is unsupported

### ISSUE THREAD

**Issue by dankmitchell** [alphagov/govuk-lint#100]
```
Error: Unknown Ruby version 2.5 found in `.ruby-version`.
Supported versions: 2.1, 2.2, 2.3, 2.4
```

**Comment by tijmenb** [comment]
Hi @danmitchell-, where are you seeing this error?

**Comment by dankmitchell** [comment]
When I run `bundle exec govuk-lint-ruby app config db lib spec --format clang`.

I did raise a PR a while ago which bumps rubocop but its gone stale now https://github.com/alphagov/govuk-lint/pulls
